mri/operators/fourier/utils.py

```diff
@@ -251,6 +252,28 @@ def gridded_inverse_fourier_transform_stack(kspace_data_sorted,
             method=method,
             fill_value=0,
         )
+    if len(idx_mask_z) < volume_shape[2]:
```

Comment:
can you explain a bit more what's happening here both in the PR description and in the docs of this function?

Also add some comments in the code about what's happening, for example: what does this test mean?

Reply:
I generally explain things in issues and add PRs trying to point to what is happening. https://github.com/CEA-COSMIC/pysap-mri/issues/81 explains pretty much what is needed.
Do you want me to repeat it here?

However, I will add about what is done and what tests are added

Reply:
Yes I think redundancy in project track-keeping is good to a certain extent.
Generally the PR indicates what was actually done compared to the first intuition presented in the issue.

Reply:
Yup, I covered that in PR. Will keep this in mind for future PRs